docs/data-access/daco/renew-and-close.md

```diff
@@ -5,52 +5,63 @@ sidebar_label: Attesting, Renewing, & Closing
+
    ![DACO renewal application](/assets/data-access/daco-renewal-application.png)
-1. The DACO Administrator will be notified that a renewal application is ready for review. If there are no issues and/or the application remains the same as the previous version, the project team should be quickly approved for an additional 2 years.
+
+3. The DACO Administrator will be notified that a renewal application is ready for review. If there are no issues and/or the application remains consistent with the previous version, the project team should be quickly approved for an additional 2 years.
 
 ### Expired and Non-renewable
 
-A signed copy of the renewal application must be sent to the ICGC DACO via the online application system. If a renewal is not received within 90 days of the application's expiry date, it will be considered closed permanently.
+A signed copy of the renewal application must be sent to the ICGC DACO via the online application system. If a renewal is not submitted within 90 days of the application's expiry date, it will be closed permanently.
+
 ![DACO renewal period is up so application is non-renewable](/assets/data-access/daco-renewal-is-up.png)
 
 ## Closing an Application
 
-1. The application can also be closed by the Applicant or by a DACO Administrator at any time within the lifecycle of the application (before or after approval). A **Close Application** button is available from within the application header bar.
+1. The application can be closed by the Applicant or by a DACO Administrator at any time within the lifecycle of the application (before or after approval). A **Close Application** button is available from within the application header bar.
+
    ![DACO close an application button](/assets/data-access/daco-close-button.png)
-1. When clicking this button, the user will be asked to confirm this action, as it cannot be undone. If the application is being closed after it has been approved, the confirmation modal will display a warning notifying the user that this action will remove access for the project team.
-   ![DACO close confirmation modal with warning](/assets/data-access/daco-close-modal.png)
-1. Once confirming this action, the application will be closed permanently. The expiry date will change to reflect the date that the application was closed. If the applicant wishes to regain access, a new application will have to be submitted for review.
-1. The applicant and all collaborators will receive an email notifying them that the application has been closed and access to ICGC Controlled Data will be removed within 24 hours.
+
+2. When clicking this button, a confirmation window will appear, asking the user to manually enter their application ID to confirm the closure.
```

Comment:
there are 2 steps to the confirmation:
1) the initial "are you sure, you will lose access" message -> user clicks "yes, close"
2) the appId input appears in the modal for final confirmation

Not sure if you feel it's useful to add both screenshots for this

Reply:
updated the text slightly, screenshot doesn't seem necessary